consumer-a/tests/getProducts.spec.ts

```diff
@@ -1,61 +1,26 @@
-import { products as mockedProducts } from "../fixtures/getProducts/200";
-import { Pact, Matchers } from "@pact-foundation/pact";
-import { resolve } from "path";
-import { getProducts } from "../src";
-
-const {like} = Matchers;
-
-const mockedProvider = new Pact({
-    consumer: "e2e Consumer A",
-    provider: "e2e Provider",
-    // port: 9292,
-    log: resolve(process.cwd(), "logs", "mockserver-integration.log"),
-    dir: resolve(process.cwd(), "pacts"),
-    logLevel: "warn",
-    spec: 2,
-})
-
-beforeAll(() => mockedProvider.setup().then(options => {
-    console.log(`http://localhost:${options.port}`);
-    process.env.API_HOST = `http://localhost:${options.port}`;
-}));
-
-afterAll(() => mockedProvider.finalize());
-
-afterEach(() => mockedProvider.verify());
-
-describe("consumer-a/getProducts()", () => {
+import { pactWith } from 'jest-pact';
+import { API, api } from "../src";
+import { productsRequest, withListOfProductsRequest } from "provider/src/endpoints";
```

Comment:
feedback:

I intentially avoided cross package sharing as i wanted to treat each package as isolated and independant. Like in a distrubuted dev team, the provider might be written in a different language, the different consumer are distributed accross different repos etc.

Reply:
jop, fully agree. As I tried to say in the description, this is a convenience thing that one might provide to some, not a requirement that has to be used by all

Reply:
> jop, fully agree. As I tried to say in the description, this is a convenience thing that one might provide to some, not a requirement that has to be used by all

okay, you mean even if the provider would be written in Java that they could nicely provide a JS package to be used for testing.